public/pages/Correlations/containers/CorrelationsContainer.tsx

```diff
@@ -468,6 +624,845 @@ export class Correlations extends React.Component<CorrelationsProps, Correlation
+          const displayedSeverities = findingsSeverity.slice(0, MAX_DISPLAY).map((severity) => {
+            const label = getSeverityLabel(severity);
+            const { background, text } = getSeverityColor(label);
+            return (
+              <EuiBadge key={severity} style={{ backgroundColor: background, color: text }}>
+                {label}
+              </EuiBadge>
+            );
+          });
+
+          const tooltipContent = (
+            <div
+              style={{
+                maxWidth: '300px',
+                maxHeight: '400px',
+                overflow: 'auto',
+              }}
+            >
+              {findingsSeverity.slice(MAX_DISPLAY).map((severity) => {
+                const label = getSeverityLabel(severity);
+                const { background, text } = getSeverityColor(label);
+                return (
+                  <div
+                    style={{
+                      display: 'flex',
+                      flexDirection: 'column',
+                      padding: '4px',
+                      width: '100%',
+                    }}
+                  >
+                    <EuiBadge key={severity} style={{ backgroundColor: background, color: text }}>
+                      {label}
+                    </EuiBadge>
+                  </div>
+                );
+              })}
+            </div>
+          );
+
+          return (
+            <span
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                alignItems: 'center',
+                gap: '4px',
+                whiteSpace: 'normal',
+                flexWrap: 'wrap',
+                width: '100%',
+              }}
+            >
+              {displayedSeverities}
+              {remainingCount > 0 && (
+                <EuiToolTip content={tooltipContent} position="top">
+                  <EuiBadge>{`+${remainingCount} more`}</EuiBadge>
+                </EuiToolTip>
+              )}
+            </span>
+          );
+        },
+      },
+      {
+        field: 'resources',
+        name: 'Resources',
+        sortable: true,
+        render: (resources: string[]) => {
+          if (!resources || resources.length === 0) return DEFAULT_EMPTY_DATA;
+          const MAX_DISPLAY = 2;
+          const remainingCount =
+            resources.length > MAX_DISPLAY ? resources.length - MAX_DISPLAY : 0;
+          const displayedResources = resources.slice(0, MAX_DISPLAY).map((resource) => {
+            return (
+              <EuiBadge style={{ backgroundColor: '#fff', border: '1px solid #d3dae6' }}>
+                {resource}
+              </EuiBadge>
+            );
+          });
+
+          const tooltipContent = (
+            <>
+              {resources.slice(MAX_DISPLAY).map((resource) => {
+                return (
+                  <div style={{ display: 'flex', flexDirection: 'column', padding: '4px' }}>
```

Comment:
Generally we avoid using custom elements unless there is not OUI component that can be used, this is because the OUI components will take care of themes, etc. styling for us